common:
    lid_topic:  "/livox/lidar"
    imu_topic:  "/livox/imu"
    
preprocess:
    lidar_type: 1                # Livox Avia LiDAR
    scan_line:  6
    blind: 1

mapping:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    fov_degree:    100
    det_range:     260.0
    extrinsic_T: [ 0.05512, 0.02226, -0.0297 ]
    extrinsic_R: [ 1, 0, 0,
                   0, 1, 0,
                   0, 0, 1]